src/containers/AppStore/components/AppCard/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, {FC, useState} from 'react';
+import {useHistory} from 'react-router';
+import {Icon, IconType} from '@thenewboston/ui';
+import * as S from './Styles';
+
+type Props = {
+  bannerUrl: string;
+  description: string;
+  id: string;
+  title: string;
+  websiteUrl: string;
+};
+
+const AppCard: FC<Props> = ({bannerUrl, description, id, title, websiteUrl}) => {
+  const [isHovered, setIsHovered] = useState(false);
+  const history = useHistory();
+
+  const handleClickCard = (): void => {
+    history.push(`/app-store/${id}`);
+  };
+
+  const handleVisitWebsite = (event: React.MouseEvent<HTMLDivElement>): void => {
+    event.stopPropagation(); // prevent from calling the click event at parent, i.e. clicking of card
+    window.open(websiteUrl, '_blank', 'noopener noreferrer');
+  };
+
+  return (
+    <S.Container role="button" onClick={handleClickCard} tabIndex={0}>
+      <S.Banner alt={title} className="AppCard__banner" loading="lazy" src={bannerUrl} />
+      <S.BottomContainer className="AppCard__bottom-container">
+        <S.DetailsContainer className="AppCard__details-container">
+          <S.AppTitleDescriptionContainer className="AppCard__app-title-description-container">
+            <S.AppTitle className="AppCard__app-title">{title}</S.AppTitle>
+            <S.AppDescription className="AppCard__app-description">{description}</S.AppDescription>
```

Comment:
what's the use of classNames if we're using styled-components?

Reply:
ah my bad, i created the `scss` version and migrated to styled-components afterwards, that's why there's some mistakes here and there, thanks for the comments